diff --git a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.dev.js b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.dev.js
index 09e1996..fc27553 100644
--- a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.dev.js
+++ b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.dev.js
@@ -43,7 +43,8 @@ const createVanillaExtractPlugin = (pluginOptions = {}) => (nextConfig = {}) =>
         isServer,
         isDevelopment: dev,
         future: nextConfig.future || {},
-        experimental: nextConfig.experimental || {}
+        experimental: nextConfig.experimental || {},
+        hasAppDir: nextConfig.experimental.appDir
       }, () => css.lazyPostCSS(dir, getSupportedBrowsers(dir, dev)), [])
     });
     config.plugins.push(new webpackPlugin.VanillaExtractPlugin({
diff --git a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.prod.js b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.prod.js
index 09e1996..fc27553 100644
--- a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.prod.js
+++ b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.cjs.prod.js
@@ -43,7 +43,8 @@ const createVanillaExtractPlugin = (pluginOptions = {}) => (nextConfig = {}) =>
         isServer,
         isDevelopment: dev,
         future: nextConfig.future || {},
-        experimental: nextConfig.experimental || {}
+        experimental: nextConfig.experimental || {},
+        hasAppDir: nextConfig.experimental.appDir
       }, () => css.lazyPostCSS(dir, getSupportedBrowsers(dir, dev)), [])
     });
     config.plugins.push(new webpackPlugin.VanillaExtractPlugin({
diff --git a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.esm.js b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.esm.js
index 83af576..d9237ad 100644
--- a/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.esm.js
+++ b/node_modules/@vanilla-extract/next-plugin/dist/vanilla-extract-next-plugin.esm.js
@@ -35,7 +35,8 @@ const createVanillaExtractPlugin = (pluginOptions = {}) => (nextConfig = {}) =>
         isServer,
         isDevelopment: dev,
         future: nextConfig.future || {},
-        experimental: nextConfig.experimental || {}
+        experimental: nextConfig.experimental || {},
+        hasAppDir: nextConfig.experimental.appDir
       }, () => lazyPostCSS(dir, getSupportedBrowsers(dir, dev)), [])
     });
     config.plugins.push(new VanillaExtractPlugin({